includes/Classifai/Admin/Settings.php

```diff
@@ -84,14 +86,27 @@ public function admin_enqueue_scripts( $hook_suffix ) {
 
 		wp_set_script_translations( 'classifai-settings', 'classifai' );
 
+		$post_types         = get_post_types_for_language_settings();
+		$excerpt_post_types = array();
+		$post_type_options  = array();
+		foreach ( $post_types as $post_type ) {
+			$post_type_options[ $post_type->name ] = $post_type->label;
+			if ( post_type_supports( $post_type->name, 'excerpt' ) ) {
+				$excerpt_post_types[ $post_type->name ] = $post_type->label;
+			}
+		}
+
 		$data = array(
-			'features'      => $this->get_features(),
-			'services'      => get_services_menu(),
-			'settings'      => $this->get_settings(),
-			'dashboardUrl'  => admin_url( '/' ),
-			'nonce'         => wp_create_nonce( 'classifai-previewer-action' ),
-			'postStatuses'  => get_post_statuses_for_language_settings(),
-			'isEPinstalled' => is_elasticpress_installed(),
+			'features'         => $this->get_features(),
+			'services'         => get_services_menu(),
+			'settings'         => $this->get_settings(),
+			'dashboardUrl'     => admin_url( '/' ),
+			'nonce'            => wp_create_nonce( 'classifai-previewer-action' ),
+			'postTypes'        => $post_type_options,
```

Comment:
This could be moved under `features`, but since we are using the `get_post_types_for_language_settings` helper function to retrieve post types, it has been kept here to continue using the same filter and avoid introducing unnecessary loop in the features data.